Bottom: c3476114911308c187afeb047fc359911f8f30f8
Top:    b8244af4f2a379a6dc0be26106cb5a19dbd12459
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-28 17:41:28 +0000

Tmp


---

diff --git a/src/polygon1.c b/src/polygon1.c
index fc7a25a..5a7a260 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1351,7 +1351,7 @@ InsertHoles (jmp_buf * e, POLYAREA * dest, PLINE ** src)
       else
 	{
           /* Need to check if this new hole means we need to kick out any old ones for reprocessing */
-          while (0) {
+          while (1) {
             struct find_inside_info info;
             PLINE *prev;
 
@@ -1371,7 +1371,7 @@ InsertHoles (jmp_buf * e, POLYAREA * dest, PLINE ** src)
             }
 
             printf ("Damnit\n");
-
+#if 1
             /* We need to find the contour before it, so we can update its next pointer */
             prev = container;
             while (prev->next != info.result) {
@@ -1384,6 +1384,7 @@ InsertHoles (jmp_buf * e, POLYAREA * dest, PLINE ** src)
             /* Add hole as the next on the list to be processed in this very function */
             info.result->next = *src;
             *src = info.result;
+#endif
           }
           /* End check for kicked out holes */
